### ISSUE TITLE
loss didn't get down

### ISSUE THREAD

**Issue by chutongz** [dontfollowmeimcrazy/imagenet#13]
Hi, I am using your code on both VGG and your original alexnet networks.
The way I manage my data is exact following your instructions, and I tried to use np_util.to_categorial() function instead, but the loss is always around 8 even after 100+ epochs, and 16 on the original code.
Do you have any idea what's the probable problem of it?
Any help is appreciated, thanks a lot!

**Comment by matteo-dunnhofer** [comment]
Hi @chutongz 
have you already tried to lower the learning rate? Like to 1e-4?

**Comment by briansune** [comment]
@dontfollowmeimcrazy i do have the same situation with your posted code. If learning rate is set to 1e-2 the loss drop logically, while after 1 epoch the loss saturate and no longer drop. May i ask what is the final settings for a workable training?
